### ISSUE TITLE
Pry/Minitest issues when using toys

### ISSUE THREAD

**Issue by halorrr** [dazuma/toys#231]
I'm having some issues using `pry` with minitest when using toys.

I'm using the build in `expand :minitest, files: ["test/**/*_test.rb"], libs: ["test", "lib"]`.

When using rake. If I had a `binding.pry` in a test, the test execution would stop at the pry line with a console I can interact with. When using toys, it prints the pry console content to the console, but then continues on without actually stopping like:

```
Run options: --seed 37978

# Running:

........................................
From: /Users/username/lunchmoney-ruby/test/lunchmoney/assets_test/asset_calls_test.rb:39 AssetCallsTest#test_assets_returns_an_array_of_Error_objects_on_error_response:

    32: test "assets returns an array of Error objects on error response" do
    33:   response = mock_faraday_response({ "error": "This is an error" })
    34: 
    35:   LunchMoney::AssetCalls.any_instance.stubs(:get).returns(response)
    36: 
    37:   api_call = LunchMoney::AssetCalls.new.assets
    38: 
 => 39:   binding.pry
    40: 
    41:   assert_kind_of(LunchMoney::Error, api_call.first)
    42: end

[1] pry(#<AssetCallsTest>)> 
.........

Finished in 0.031749s, 1543.3557 runs/s, 2456.7703 assertions/s.
49 runs, 78 assertions, 0 failures, 0 errors, 0 skips
```

This is the project I'm currently running toys on and running into this:
https://github.com/halorrr/lunchmoney-ruby and resetting to commit `6f4dc6c67b8d4e5820cd8c08550d5981831935da` would be before I migrated from rake to toys if you want to compare how `rake test` behaves.

Is this a bug in toys? Have I misconfigured something?

**Comment by dazuma** [comment]
Thanks for the report. Looks like this is happening because the Toys minitest integration (unlike Rake's testtask) takes over the input stream, not expecting that someone might want to interact with the process during a test run. I can't think of a good workaround, unfortunately, but I think it should be an easy fix.

**Comment by dazuma** [comment]
This is fixed at HEAD now. I'm working on one more fix, and expect to release this in version 0.15 early next week.
